plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/_core_aws.py

```diff
@@ -0,0 +1,182 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
```

Comment:
this entire file shows up as one big change.
what's new here?

Reply:
Nothing new, I just moved it from ray_laucher.py to its seperate file